.lgtm.js

```diff
@@ -0,0 +1 @@
+function appease_lgtm() {}
```

Comment:
Possibly a silly question, but do we care if LGTM analysis fails for javascript? It fails for all the other lanugages F' doesn't use